engine/core/src/main/java/org/datacleaner/descriptors/AbstractMethodDescriptor.java

```diff
@@ -70,22 +74,29 @@ public final String toString() {
         return getClass().getSimpleName() + "[method=" + _name + "]";
     }
 
-    protected final void invoke(Object component) {
+    protected final void invoke(Object component) throws RuntimeException, IllegalStateException {
```

Comment:
Well yes and no :) The RuntimeException is basically thrown if anything goes wrong in the validation method itself. The IllegalStateException is explicitly mentioned also because the code here (the "wrapper" around the validation method) might also throw such an exception.